docs/coding-guidelines/breaking-change-rules.md

```diff
@@ -45,6 +45,12 @@ Breaking Change Rules
 
 * Changing the precision of a numerical return value
 
+* Removing a finalizer from an unsealed public type once it has one
```

Comment:
I do not think that this is correct statement. It should be fine to remove finalizer as long as the resource cleanup that the finalizer was responsible for is done in some other way, for example it can be done by a different type with the same lifetime.

Reply:
This is what @bartonjs said in the email thread citing the Framework Design Guidelines, 3rd edition (section 9.4, p387).

Reply:
(2nd attempt) The guideline is essentially assuming that the finalizer body is just `Dispose(false)` in conjunction with the basic dispose pattern.

Perhaps "Removing a `Dispose(false)` finalizer from an unsealed public type once it has one"?

Reply:
Yes, that sounds more accurate.

It is a specialization of more general "Removing a call to user-overridable virtual method." that is a common gotcha.

Reply:
> It is a specialization of more general "Removing a call to user-overridable virtual method." that is a common gotcha.

Exactly.  There are plenty of places we've had to avoid removing (or adding or changing in an observable way) calls to an overridable virtual.  A finalizer that calls a virtual Dispose is just one of them.

Reply:
It looks like this document doesn't call out 
> removing a call to user-overridable virtual method

So perhaps we should add that instead, or in addition.

It seems odd to me that this is under `Property, Field, Parameter and Return Values` heading.  Perhaps we need a new heading for `Methods` or we should change this to `Members`